node/cli_tools.go

```diff
@@ -67,7 +68,7 @@ func FlagsToArgs(flags Flags, basedir string, sepBase bool) ([]string, Metadata)
 		"--bootstrap-ips=" + flags.BootstrapIPs,
 		"--bootstrap-ids=" + flags.BootstrapIDs,
 		"--bootstrap-beacon-connection-timeout=" + flags.BootstrapBeaconConnectionTimeout,
-		"--db-enabled=" + strconv.FormatBool(flags.DBEnabled),
+		"--db-type=" + dbType,
```

Comment:
This needs to be `flags.DBType`, so that if a user specifies the database type, it will be passed in correctly.